include/Macro.h

```diff
@@ -528,13 +539,15 @@
 // particle type macros
 
 // number of particle types (default: 4)
-#  define  PAR_NTYPE                4
+#  define  PAR_NTYPE                8
 
 // particle type indices (must be in the range 0<=index<PAR_NTYPE)
-#  define  PTYPE_TRACER          (real_par)0
-#  define  PTYPE_GENERIC_MASSIVE (real_par)1
-#  define  PTYPE_DARK_MATTER     (real_par)2
-#  define  PTYPE_STAR            (real_par)3
+#  define  PTYPE_TRACER          (real)0
+#  define  PTYPE_GENERIC_MASSIVE (real)1
+#  define  PTYPE_DARK_MATTER     (real)2
+#  define  PTYPE_STAR            (real)3
+#  define  PTYPE_CEN             (real)4
```

Comment:
What are `PTYPE_CEN` and `PTYPE_CLUSTER`?

Reply:
IIUC, `PTYPE_CLUSTER+c` is the particles of the halo `c`, and `PTYPE_CEN+c` is the BH particle of the halo `c`, where `c` should be `0` and `1` in this PR.